platform/spec/services/temporal/activities/update_connection_status_activity_spec.rb

```
@@ -0,0 +1,95 @@
+      it 'raises ArgumentError' do
+        expect {
+          subject.execute(
+            connection_id: connection.id,
+            status: :invalid_status
+          )
+        }.to raise_error(ArgumentError, /Invalid status/)
+      end
+    end
+  end
+
+  describe 'retry policy' do
```

rubocop (1.68.0)

[convention] Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.

(Style/StringLiterals)